payments-model/src/test/java/com/stripe/android/model/CardPaymentDetailTest.kt

```diff
@@ -0,0 +1,84 @@
+package com.stripe.android.model
+
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.core.model.CountryCode
+import org.junit.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+
+class CardPaymentDetailTest {
+
+    private val card = ConsumerPaymentDetails.Card(
+        id = "QAAAKIL",
+        last4 = "4242",
+        expiryYear = 2500,
+        expiryMonth = 4,
+        brand = CardBrand.Visa,
+        cvcCheck = CvcCheck.Fail,
+        isDefault = false,
+        billingAddress = ConsumerPaymentDetails.BillingAddress(
+            countryCode = CountryCode.US,
+            postalCode = "42424"
+        )
+    )
+
+    @Test
+    fun `isExpired should be true when expiry date is in the past`() {
+        assertThat(card.copy(expiryYear = 2005).isExpired).isTrue()
+    }
+
+    @Test
+    fun `isExpired should be false when expiry date is in the past`() {
```

Comment:
is the name of this test correct? It doesn't look the expiry date is in the past here

Reply:
No, just fixed that. Thanks for catching that.